guides/developer/getting-set-up.md

```diff
@@ -118,3 +118,11 @@ If you haven't done so already, read through the
 services' responsibilities and their dependencies on one-another. The
 [list of ports each Spinnaker service uses](https://www.spinnaker.io/reference/architecture/#port-mappings)
 can be very useful when querying and debugging a service's API.
+
+Consider working on one of the
+[issues marked "beginner-friendly"](https://github.com/spinnaker/spinnaker/issues/3065) to start
```

Comment:
Why not link to the list of beginner friendly issues? (https://github.com/spinnaker/spinnaker/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22)

Reply:
If there's no objection I'll open a PR with this change

Reply:
This was a copy/paste error.  fixed here: https://github.com/spinnaker/spinnaker.github.io/pull/921